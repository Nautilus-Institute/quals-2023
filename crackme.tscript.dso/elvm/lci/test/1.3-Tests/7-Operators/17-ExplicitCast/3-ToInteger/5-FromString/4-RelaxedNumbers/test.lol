HAI 1.3
	VISIBLE MAEK " 123" A NUMBR BTW leading whitespace is ignored
	VISIBLE MAEK "123x" A NUMBR BTW parsing stops at the first non-digit
	VISIBLE MAEK "0377" A NUMBR BTW octal prefix
	VISIBLE MAEK "0xFF" A NUMBR BTW hexadecimal prefix
KTHXBYE
