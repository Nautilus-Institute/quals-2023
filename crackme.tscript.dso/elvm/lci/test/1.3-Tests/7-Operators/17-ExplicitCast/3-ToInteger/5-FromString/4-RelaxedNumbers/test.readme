This test checks that the explicit cast operator correctly casts string values
to integer types as would be done by libc's strtoll() function.
