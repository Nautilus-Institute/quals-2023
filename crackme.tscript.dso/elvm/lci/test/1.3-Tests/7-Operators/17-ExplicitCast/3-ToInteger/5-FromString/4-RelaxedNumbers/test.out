123
123
255
255
